src/gui/window_file_list.c

```diff
@@ -142,31 +66,27 @@ void window_file_list_draw(window_file_list_t *window) {
     int visible_count = rc_win.h / item_height;
     int i;
     for (i = 0; i < visible_count && i < window->count; i++) {
-        int idx = i + window->top_index;
-        uint16_t id_icon = IDR_NULL;
-        if (idx == 0) {
-            id_icon = (!strcmp(window->altpath, "/")) ? IDR_PNG_filescreen_icon_home : IDR_PNG_filescreen_icon_up_folder;
-        } else if (window->file_items[idx].fattrib & AM_DIR) {
-            id_icon = IDR_PNG_filescreen_icon_folder;
+        bool isFile = true;
+        const char *item = LDV_FileAt(window->ldv, i, &isFile);
```

Comment:
`LDV_FileAt` returns `nullptr` in case there is no file at the position. 
I believe it does not „usually happen“ here, but I also think it will happen in the case of `firstDirEntry` being set to a file „at the of the directory“. Am I right?